### ISSUE TITLE
View location on a map

### ISSUE THREAD

**Issue by barnesjt** [Barnesjt/appstudio-weather#3]
Need to be able to view the location on a map

**Comment by barnesjt** [comment]
This one I had a bit more trouble messing around with yesterday (I believe it was just a small syntax error that held me up), but I did get something that works.

Here's some relevant documentation:
https://developers.arcgis.com/qt/qml/api-reference/qml-esri-arcgisruntime-mapview.html
https://developers.arcgis.com/qt/qml/api-reference/qml-esri-arcgisruntime-map.html
https://developers.arcgis.com/qt/qml/api-reference/qml-esri-arcgisruntime-basemapimagery.html
https://developers.arcgis.com/qt/qml/api-reference/qml-esri-arcgisruntime-viewpointcenter.html
https://developers.arcgis.com/qt/qml/api-reference/qml-esri-arcgisruntime-pointbuilder.html

Here's a page that I was able to put together yesterday, which shows a map of the current location. Ideally, I would like to add some marker of the current location, but might not be really needed.

```
Page {
  id: locationview
  signal back()
  clip: true

  property real longitude
  property real latitude

  PointBuilder {
    id: buildPoint
    spatialReference: SpatialReference { wkid: 4326 }
  }

  header: ToolBar {
    RowLayout {
      anchors.fill: parent
      ToolButton {
        text: qsTr("<")
        onClicked: back()
      }
    }
  }

  Rectangle {
    anchors.fill: parent
    color: app.primaryColor

    MapView{
      id: mapView
      anchors.fill: parent
      Map {
        BasemapImagery {}
        initialViewpoint: currViewpoint
      }
      ViewpointCenter {
        id: currViewpoint
        center: buildPoint.geometry
        targetScale: 15000
      }
      Component.onCompleted : {
        forceActiveFocus();
        locationDisplay.positionSource.start();
        buildPoint.setXY(longitude, latitude);
        mapView.setViewpointCenterAndScale(buildPoint.geometry, 3000.0);
      }
    }
  }
}
```

**Comment by barnesjt** [comment]
Resolved in PR #8
